relay-server/src/services/store.rs

```diff
@@ -721,7 +721,8 @@ impl StoreService {
         )?;
         metric!(
             counter(RelayCounters::ProcessingMessageProduced) += 1,
-            event_type = "metric"
+            event_type = "metric",
+            namespace = namespace.as_str()
```

Comment:
Please document this tag here: https://github.com/getsentry/relay/blob/fbedc4878702ea405c6f741a07917a65da58a129/relay-server/src/statsd.rs#L540-L543

Reply:
thanks! i did